app/authentication/no_survey_data_to_flush.py

```diff
@@ -0,0 +1,7 @@
+class NoSurveyDataToFlush(Exception):
```

Comment:
This exception could be named something more generic, such as SurveyDataException, in order to prevent further exceptions from being created and attached to the same view.